src/EIPTestsFiller/StateTests/stEOF/stEIP3540/CREATE2_EOF1_FromEOFFiller.yml

```diff
@@ -30,14 +30,10 @@ CREATE2_EOF1_FromEOF:
     data: 
       # Deploying codes starting with 0xef
 
-      #- ':yul { mstore8(0, 0xef) return(0, 1) }'
-      - ':label deploying_0xef :raw 0x60ef60005360016000f3'
-      #- ':yul { mstore8(0, 0xef) return(0, 2) }'
-      - ':label deploying_0xef00 :raw 0x60ef60005360026000f3'
-      #- ':yul { mstore8(0, 0xef) return(0, 3) }'
-      - ':label deploying_0xef0000 :raw 0x60ef60005360036000f3'
-      #- ':yul { mstore8(0, 0xef) return(0, 32) }'
-      - ':label deploying_0xef00..00 :raw 0x60ef60005360206000f3'
+      - ':label deploying_0xef :yul-eof { mstore8(0, 0xef) return(0, 1) }'
```

Comment:
what is yul-eof let me add it to the config

Reply:
Here is the yul-eof script: https://github.com/hugo-dc/retesteth-config/blob/eof/default/yul-eof.sh
It's using solidity from this branch: https://github.com/ethereum/solidity/tree/eof-functions